torch/optim/optimizer.py

```diff
@@ -67,6 +67,27 @@ def _default_to_foreach(tensorlists: List[List[torch.Tensor]], differentiable: b
     )
 
 
+# Common doc strings among optimizers
```

Comment:
nit: these should most likely be private ;)

Reply:
done

Reply:
i did not bother reordering imports